docker/tinode/entrypoint.sh

```diff
@@ -140,10 +120,10 @@ args=("--config=${CONFIG}" "--static_data=$STATIC_DIR")
 
 # Maybe set node name in the cluster.
 if [ ! -z "$CLUSTER_SELF" ] ; then
-  args+=("--cluster_self=$CLUSTER_SELF")
+	args+=("--cluster_self=$CLUSTER_SELF")
 fi
 if [ ! -z "$PPROF_URL" ] ; then
-  args+=("--pprof_url=$PPROF_URL")
+	args+=("--pprof_url=$PPROF_URL")
 fi
 
 # Create the log directory (/var/log/tinode-`current timestamp`).
```

Comment:
I'm not sure we need to setup log rotation here. I would just write to log file and be done with it. Users may have their own logging needs and methods.

Reply:
Removed.